index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/0278cc9d4c.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css">
+    <title>John Doe</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <a href="#" class="navbar__logo">
+                John Doe
+            </a>
+            <ul class="navbar__menu">
+                <li>
+                    <i class="fa-solid fa-bars"></i>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <section class="header-section">
+        <h1 class="heading__primary">Hey There.
+            I’m Alex
+        </h1>
+        <h2 class="heading__secondary">
+            I’m a Software Developer
+        </h2>
+        <p class="heading__about">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.
+        </p>
+        <ul class="header-section__social">
+            <li class="header-section__icon">
+                <i class="fa-brands fa-github"></i>
+            </li>
+            <li class="header-section__icon">
+                <i class="fa-brands fa-linkedin-in"></i>
+            </li>
+            <li class="header-section__icon">
+                <i class="fa-brands fa-angellist"></i>
+            </li>
+            <li class="header-section__icon">
+                <i class="fa-brands fa-twitter"></i>
+            </li>
+            <li class="header-section__icon">
+                <i class="fa-brands fa-medium-m"></i>
+            </li>
+            
+            </ul>
```

Comment:
- [ ] Kindly add an a tag to each list icon, each li should look like this: 
`            <li><a href="#"><i class="fa-brands fa-medium-m"></i></a></li>
`